extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -319,6 +337,27 @@ export class AmpSelector extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Handles selectUp events.
+   * @param {number} incrementPos
```

Comment:
bikeshed: maybe rename this `delta`, since it could be an increment or a decrement